public_html/profiles/social/modules/social_features/social_search/src/Form/SearchContentForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\social_search\Form\SearchContentForm.
+ */
+
+namespace Drupal\social_search\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\Component\Utility\UrlHelper;
+
+/**
+ * Class SearchContentForm.
+ *
+ * @package Drupal\social_search\Form
+ */
+class SearchContentForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'search_content_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+
+    $form['search_input'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Search the entire website'),
+    );
+    // Prefill search input if we are on search content page.
+    if (\Drupal::routeMatch()->getRouteName() == 'view.search_content.page') {
+      // As function arg() is deprecated this is the only way to retrieve
+      // an array which contains the path pieces.
+      // See: https://www.drupal.org/node/2274705
+      $current_path = \Drupal::service('path.current')->getPath();
+      $path_args = explode('/', $current_path);
+      if(!empty($path_args[3])){
```

Comment:
@ribel Is there a way to retrieve the used argument (3) from the view programmatically? If developers now change the path of the view to something else this code will not work anymore.